addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/UserControlledOpenRedirectScanner.java

```diff
@@ -119,7 +132,14 @@ private void checkUserControllableLocationHeaderValue(
                     || paramValue.equalsIgnoreCase(token)
                     || (responseLocation.indexOf("://") > 0
                             && paramValue.indexOf(responseLocation) >= 0)) {
-                raiseAlert(msg, id, param.getName(), paramValue, responseLocation);
+                // requestDomain is null if some exception occurred. if requestDomain is equal to
+                // location header then it is not considered as an issue.
+                // https://github.com/zaproxy/zaproxy/issues/5289
```

Comment:
For my 2 cents this comment isn't necessary. The variables are instantiated null, so it's obvious the code is checking that, and that the domain(s) aren't the same.

Reply:
removed the comments. Resolving conversation.